modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/upgrade-catch-all-check/LPD_48918.testjava

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Regisson Aguiar
+ */
+@Component(
+	immediate = true,
+	property = {
+		"commerce.shipping.engine.key=" + CustomCommerceShippingEngine.KEY,
+		"service.ranking:Integer=100"
+	},
+	service = CommerceShippingEngine.class
+)
+public class LPD_48918 implements CommerceShippingEngine {
+
+	public static final String KEY = "by-weight";
+
+	@Override
+	public String getCommerceShippingOptionLabel(String name, Locale locale) {
+
+		getCommerceShippingOptionLabel(name,locale);
+
+	}
+
+	@Override
+	public List<CommerceShippingOption> getCommerceShippingOptions(
+			CommerceContext commerceContext, CommerceOrder commerceOrder,
+			Locale locale)
+		throws CommerceShippingEngineException {
+
+		getCommerceShippingOptions(commerceContext,commerceOrder,locale);
+
+	}
+
+	@Override
+	public String getDescription(Locale locale) {
+
+		getDescription(locale);
+
+	}
+
+	@Override
+	public String getKey() {
+		return KEY;
+	}
+
+}
```

Comment:
@regiCesar21 remember the rules for the previous patterns tests, only the minimum necessary and relevant for the test should be on the test file. So, could be reduced 👍🏻 

```
@Component(
	immediate = true,
	property = {
		"commerce.shipping.engine.key=" + CustomCommerceShippingEngine.KEY,
		"service.ranking:Integer=100"
	},
	service = CommerceShippingEngine.class
)
public class LPD_48918 implements CommerceShippingEngine {

	public static final String KEY = "by-weight";

	@Override
	public String getKey() {
		return KEY;
	}

}
```